### DISCUSSION TITLE
Borg -Failed to create/acquire the lock /root/.cache/borg/xxxxxxxxxx.lock.exclusive

### DISCUSSION THREAD

**Discussion by Aceriz** [nextcloud/all-in-one#2424] [Questions] [answered]
Okay so I have made some progress where I am decided to just go with the above.   This now has the nextcloudaio Mastercontainer starting up. 

 

But when I try to run the backup I am getting error ===> Failed to create/acquire the lock /root/.cache/borg/xxxxxxxxxx.lock.exclusive "   
 Failed to create/acquire the lock" and point to a lock.exclusive file in my backup folder.

 

Steps to replicate it:

Manually start a backup

Additional Info:

This started after a crash during an update to the mastercontainer which I have now remedied. 

From within the Mastercontainer Interface I can run the check backup integrity command.  Which comes back clean without reported errors. 

From what I can find it looks like I need to run the borg break-lock  Command  https://borgbackup.readthedocs.io/en/stable/usage/lock.html#borg-break-lock

I can not seem to figure out how to run this in Unraid..  as the "nextcloud-aio-borgbackup"  docker closes immediately after about 2 second when I attempt to run the backup so not enough time to activate anything... Nor do I really know what the command would be..   is the command ==>   borg -v break-lock /mnt/user/Backups/nextcloudaio\ Backup/borg
?

 

thanks for help

**Comment by szaimen** [comment, marked as answer]
Hi, can you check if this helps?
https://github.com/nextcloud/all-in-one/discussions/2350#discussioncomment-5742385

**Reply by Aceriz** [reply]
> Hi, can you check if this helps? [#2350 (comment)](https://github.com/nextcloud/all-in-one/discussions/2350#discussioncomment-5742385)

OMG !! it worked THANKS SO MUCH !!!!
